GuillotineMenu.podspec

```diff
@@ -12,10 +12,10 @@ Pod::Spec.new do |spec|
   spec.platform = :ios, '8.0'
   spec.ios.deployment_target = '8.0'
 
-  spec.source = { :git => "https://github.com/Yalantis/GuillotineMenu.git", :tag => “2.0.1” }
+  spec.source = { :git => "https://github.com/joekr/GuillotineMenu.git", :tag => "2.0.2" }
```

Comment:
I didn't mean to for this PR. I had to test it, and I guess I committed it :confounded: I will fix this